src/logger.cpp

```diff
@@ -0,0 +1,87 @@
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <locale>
+#include <sstream>
+
+#include "logger.h"
+
+namespace ohl::logger {
+
+const char* logPath = "ohl.log";
```

Comment:
Prefer `std::[w]string` to `[w]char*`, it should always implicitly cast. In this case, since it's a path, prefer `std::filesystem::path`.

Additionally, I think this should be relative to the dll, not the cwd. `loader.cpp` has an example of how to do so, though since two places need the path now it probably makes more sense to extract that as part of `startup_thread`, and pass the dll path to each module's init.